vl-kontrakt-fp-formidling/src/main/java/no/nav/foreldrepenger/kontrakter/fpsak/beregningsgrunnlag/v2/kodeverk/OpptjeningAktivitetDto.java

```diff
@@ -12,12 +12,13 @@ public enum OpptjeningAktivitetDto {
     OPPLÆRINGSPENGER,
     PLEIEPENGER,
     FRISINN,
+    UNG,
```

Comment:
Usikker på om denne skal være opptjeningsaktivitet

Reply:
Vi kan fjerne den seinare om den ikkje skal med. Eg trur den avklaringa ligg litt lenger fram i tid.

Reply:
Ta den når det er avklart - eller i det minste kommenter. Selv om tema er UNG så skriv navnet helt ut her.
Tenk vedlikeholdbarhet - ellers ligger dette som et åpent spørsmål om et par år.

Reply:
Da har jeg fjernet UNG herfra, så får vi håndtere det senere dersom ytelsen skal inngå som opptjening for FP/SVP